packages/core/src/account.ts

```diff
@@ -227,6 +227,25 @@ export class Account extends PBES2Container implements Storable {
         this._key = account._key;
     }
 
+    validateOrThrowInputValues() {
+        const NAME_MAX_LENGTH = 100;
+        const EMAIL_MAX_LENGTH = 255;
```

Comment:
There's no _great_ reason for me to set these values. The numbers just seemed sensible to me, probably from decades-old indoctrination about database limits that are irrelevant now.